### PR TITLE
added release notes for 10.16.0.383

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-383.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device management</td>
+<td>Availability calculations in the "Availability" overview and for individual devices were not correct. Percentages were calculated for the needed time plus an additional 24 hours. The time interval has now been corrected.</td>
```
Comment (rssag) [inline content/release-10-16-0/devices-10-16-0-bundle/10-16-0-383.md:26]:
```suggestion
<td>Availability calculations in the "Availability" overview and for individual devices were not correct. Percentages were calculated for the needed time plus an additional 24 hours. The time interval has now been corrected.</td>
```

what's the correct interval?

maybe also replace "needed" with "required"

File: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-383.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+<td>DM-2737</td>
+</tr>
+
+<tr>
+<td>Device management</td>
+<td>If editing a device protocol details page and navigating out of that page, a confirmation popup is now always properly shown.</td>
```
Comment (rssag) [inline content/release-10-16-0/devices-10-16-0-bundle/10-16-0-383.md:32]:
```suggestion
<td>After editing a device protocol details page and leaving the page, a confirmation pop-up is now always properly shown.</td>
```

File: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-383.md (hunk 1 above)
Comment (BeateRixen) [inline content/release-10-16-0/devices-10-16-0-bundle/10-16-0-383.md:26]:
```suggestion
<td>The availability calculations in the "Availability" overview and for individual devices were not correct. Percentages were calculated for the respective time (24 hours, last 7 days and last 30 days) plus an additional 24 hours. The time interval has now been corrected.</td>
```

Comment (BeateRixen) [reply content/release-10-16-0/devices-10-16-0-bundle/10-16-0-383.md]: Changed to be more clear

File: content/release-10-16-0/applications-10-16-0-bundle/10-16-0-383.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+<tr>
+<td>Cockpit</td>
+<td>The tooltip in the "Linear gauge" widget no longer overflows the widget card. Moreover, the tooltip color changes according to the type of the declared range (default, red or yellow).</td>
+<td>MTM-55005</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>n rare cases, when the real-time connection was interrupted, the UI did not resubscribe to the previously subscribed channels once the connection was re-established. This behavior has been fixed.</td>
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/release-10-16-0/applications-10-16-0-bundle/10-16-0-383.md:32]: Thanks, I will fix it directly in develop/main

**Pull request merged**